src/tree/annotation.rs

```diff
@@ -0,0 +1,116 @@
+use canonical::{Canon, Store};
+use canonical_derive::Canon;
+use core::borrow::Borrow;
+use dusk_plonk::prelude::BlsScalar;
```

Comment:
It's just a minor thing, but should't prelude be always fully imported? Otherways, you should get the scalar from the dusk_plonk::bls12_381::Scalar as BlsScalar.

Reply:
Fixed